soc/st/stm32/stm32wbax/soc.c

```diff
@@ -38,6 +38,9 @@ int stm32wba_init(void)
 	LL_ICACHE_SetMode(LL_ICACHE_1WAY);
 	LL_ICACHE_Enable();
 
+#if defined(CONFIG_FLASH_PREFETCH)
+	__HAL_FLASH_PREFETCH_BUFFER_ENABLE();
```

Comment:
Please use LL

Reply:
Only HAL macro definition is for now available in the libraries, no LL equivalent right now

Reply:
ok, let's go with this version for now, but an issue should be raised on the LL then.